composeApp/src/commonMain/kotlin/App.kt

```diff
@@ -1,25 +1,31 @@
-import androidx.compose.animation.AnimatedVisibility
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.lazy.LazyColumn
-import androidx.compose.foundation.lazy.items
-import androidx.compose.material.Button
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Text
 import androidx.compose.runtime.*
-import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import org.example.zoodbia.ui.main.MainViewModel
-import org.example.zoodbia.di.koinViewModel
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import org.example.zoodbia.navigation.AppNavigation
+import org.example.zoodbia.navigation.SplashNav
 import org.jetbrains.compose.ui.tooling.preview.Preview
-import org.koin.compose.KoinContext
 
 @Composable
 @Preview
 fun App() {
     MaterialTheme {
-        KoinContext {
-            Navigation()
+        val navigator = rememberNavController()
+
```

Comment:
I would not move this method, for cleaners and code separation

Reply:
Resolved